Sources/ComposableArchitecture/Scheduling/TestScheduler.swift

```diff
@@ -82,11 +79,16 @@ where SchedulerTimeType: Strideable, SchedulerTimeType.Stride: SchedulerTimeInte
     options _: SchedulerOptions?,
     _ action: @escaping () -> Void
   ) {
-    self.scheduled.append((UUID(), date, action))
+    self.scheduled.append((self.nextId(), date, action))
   }
 
   public func schedule(options _: SchedulerOptions?, _ action: @escaping () -> Void) {
-    self.scheduled.append((UUID(), self.now, action))
+    self.scheduled.append((self.nextId(), self.now, action))
+  }
+
+  private func nextId() -> Int {
+    self.lastId += 1
```

Comment:
Should we care about overflow ever? Probably unlikely. Could use `UInt` to double the number of work that can be scheduled?

Reply:
done, but yeah, I bet Swift couldn't even execute that much work in a reasonable amount of time 😂